src/panels/lovelace/cards/hui-alarm-panel-card.ts

```diff
@@ -294,10 +294,15 @@ class HuiAlarmPanelCard extends LitElement implements LovelaceCard {
 
       #keypad mwc-button {
         margin-bottom: 5%;
+        margin-left: 1%;
+        margin-right: 1%;
```

Comment:
Remove all the margin and add less padding:
```
        padding: calc(var(--base-unit) / 3);
```

Reply:
I think we should drop that `--base-unit` var, as it doesn't do much? And I see there is a hard `300px` width on the `div#keypad`, that should go, should be `100%`

Reply:
@bramkragten I will take a look at it